src/main/antlr4/imports/mysql_literal_tokens.g4

```diff
@@ -192,6 +192,7 @@ STORED: S T O R E D;
 SUBPARTITION: S U B P A R T I T I O N;
 SUBPARTITIONS: S U B P A R T I T I O N S;
 TABLE: T A B L E;
+TABLES: T A B L E S;
```

Comment:
this file is autogenerated from `generate_tokens.rb`, plz add this token there

Reply:
ok, I kept it consistent

Reply:
everything ok like this?